tests/fast-integration/skr-test/test.js

```diff
@@ -50,21 +54,26 @@ describe('SKR test', function() {
       );
       shoot = await getSKRConfig(instanceID);
     } else {
+      debug("Provisioning new SKR instance...")
```

Comment:
nit: Why not take it as info log level instead of debug ? I think this is useful information :) Same for the other debug logs 🤔

Reply:
agree some log could be made to info level which show important state transitions. like SKR is provisioned added to runtime scenario and Provisioning of new SKR

Reply:
updated